### ISSUE TITLE
Unread message indicator does not disappear after some time has passed

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#4718]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. As user A send some messages to user B
2. As user B navigate to the conversation with user A
3. Wait a minute

## Expected Result:
Unread message indicator should disappear after: 
- Some time has passed
- The user sends a message

## Actual Result:
Unread message indicator doesn't go away after the user sends a message or some time has passed. 

## Workaround:
To remove the unread message indicator the user has to navigate away from the conversation. 

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.0.86-2

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

![image](https://user-images.githubusercontent.com/44479856/129818448-a1b997f8-e632-4174-9d28-ba407b6df623.png)

**Expensify/Expensify Issue URL:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

____
From @mallenexpensify https://expensify.slack.com/archives/C01GTK53T8Q/p1629245998499400
> On web, staging, the new message indicator is stuck and is not disappearing.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @MariaHCD (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by isagoico** [comment]
According to this PR https://github.com/Expensify/App/pull/1570, the unread indicator should disappear if: 
1. The indicator fades out after a few seconds
2. If another message is sent while the report is active, verify the indicator doesn't show up again.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @JmillsExpensify (`External`), see https://stackoverflow.com/c/expensify/questions/8582 for more details.

**Comment by Santhosh-Sellavel** [comment]
I reported this issue while discussing #4472 Improvement. In this slack thread https://expensify.slack.com/archives/C01GTK53T8Q/p1628783620254300?thread_ts=1628782470.250300&cid=C01GTK53T8Q

I'll pick this up!

**Comment by JmillsExpensify** [comment]
Sounds great! Upwork job is here: https://www.upwork.com/jobs/~01e0bdbb422755f671. Make sure to repeat your proposal here and it can be reviewed by the team.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @AndrewGable (`Exported`), see https://stackoverflow.com/c/expensify/questions/7972 for more details.

**Comment by Santhosh-Sellavel** [comment]
@JmillsExpensify I'm working on it

**Comment by MTN718** [comment]
# Proposal (Shane Watson from Upwork)

1. when hide marker, isMarkerActive is false,  in other word, scroll at the end, reset new marker position.
2. Add event when start typing user in the input, reset new marker position

**Comment by parasharrajat** [comment]
@MTN718 As per the contribution guidelines new contributors can only work on one job at a time. I just thought of telling you this as it will save you time.

**Comment by MTN718** [comment]
Thanks for your notice

**Comment by Santhosh-Sellavel** [comment]
## Proposal 
Just to hide the unread message indicator when the user posts a reply. We could just reset the marker position in the below lines.
https://github.com/Expensify/App/blob/bdcafa3444f3839d7f5de2300e258caf681a50e3/src/pages/home/report/ReportActionsView.js#L192-L194

Awaiting confirmation from @MariaHCD 
Do we need to disappear the unread indicator after some time?

**Comment by MariaHCD** [comment]
@Santhosh-Sellavel In an existing conversation where two people are chatting, I think it would make sense to hide the unread indicator after a certain time has passed. What would be the optimal time to wait until the indicator is hidden? cc: @mallenexpensify @JmillsExpensify @AndrewGable

**Comment by MelvinBot** [comment]
@JmillsExpensify, @AndrewGable Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by mallenexpensify** [comment]
Taking this over as the Contributor Manager for Mills. 
> Do we need to disappear the unread indicator after some time?

Yes, I looked around to see how long that might have been but can't find, my hunch is that it should disappear after 2-3 seconds.  

Also, [as mentioned in #expensify-open-source,](https://expensify.slack.com/archives/C01GTK53T8Q/p1629387832034200) we have some competing priorities that'll likely continue through the week so @AndrewGable might not be available to review for a few days.

**Comment by AndrewGable** [comment]
How long does Slack's notification indicator appear for?

**Comment by Santhosh-Sellavel** [comment]
I don't know I waited a minute scrolled up & down it doesn't disappear. After switching to a different chat it disappeared.

**Comment by parasharrajat** [comment]
I don't think we have to do this ATM or it is such a priority thing yo do. just throwing my 2️⃣ cents.

**Comment by Santhosh-Sellavel** [comment]
<img width="792" alt="Screenshot 2021-08-24 at 4 52 20 AM" src="https://user-images.githubusercontent.com/85645967/130531736-41241390-2989-4117-bfd8-4e3301130aeb.png">

It does not get removed after posting a reply too!

**Comment by AndrewGable** [comment]
@mallenexpensify any thoughts on this after some research on how slack does this?

**Comment by mallenexpensify** [comment]
I checked on a couple Slack convos, it looks like 4 seconds. but... it doesn't 'go away'.  For the first 4 seconds is bolder/thicker
![image](https://user-images.githubusercontent.com/22508304/130653486-0b8677bb-bc73-449b-aef2-dd93c264478c.png)
then it's thinner but still red
![image](https://user-images.githubusercontent.com/22508304/130653534-fa43470d-a90d-4cc7-be0a-7f06b028e194.png)
(I tried to grab a vid with no luck). 

The line persists until you go to another chat which seems to mirror what we do in NewDot.  

I'm leaning towards @parasharrajat thought
> I don't think we have to do this ATM or it is such a priority thing to do. just throwing my 2️⃣ cents.

@AndrewGable you agree? If so, we can close
